about.html

```diff
@@ -0,0 +1,282 @@
+              placeholder="Enter email here .."
+              name="email"
+              required
+            />
+
+            <label for="register-password"><b>Password</b></label>
+            <input
+              type="password"
+              id="register-password"
+              maxlength="10"
+              minlength="8"
+              placeholder="Enter password here .."
+              name="password"
+              required
+            />
+            <div class="remember-me">
+              <input type="checkbox" checked="checked" name="remember" />
+              Remember me
+            </div>
+            <div id="error-message2"></div>
+            <button class="sign-up-submit-btn form-submit-btn" type="submit">
+              Submit
+            </button>
+            <div>
+              Have an acount
+              <span class="sign-up-in-option-btn" id="sign-in-btn3"
+                ><u><i>Login</i></u>
+              </span>
+            </div>
+          </div>
+          <div class="container form-buttom-container">
+            <button type="button" id="cancel-btn2" class="cancel-btn">
+              Cancel
+            </button>
+            <span class="psw">Forgot <a href="#">password?</a></span>
+          </div>
+        </form>
+      </div>
+    </section>
+    <!-- page main section -->
+    <main>
+      <section class="about-section">
+        <h2>
+          CHOGM <i class="bi bi-dot"></i> 2022<br />
+          RWANDA - HOSTS THE WORLD
+        </h2>
+        <p>
+          CHOGM is an annual conference that is centered around developing favorable global policies. This year it is hosted in Rwanda with lots
+          of key speakers and delegates from around the globe. This conference is a great opportunity to learn about the global issues and how to
+          address them.
+        </p>
+        <div>
+          Please contact us via Email for feedback or any other questions about
+          CHOGM 2022! <br />
+          <b>chogmrwanda2022@chogm.com</b>
+        </div>
+      </section>
+      <section class="recent-event-section">
+        <h3>See recent speakers at CHOGM</h3>
+        <div class="horizontal-line"></div>
+        <p>
+          Take a look at three top past events which were streamed live here...
+        </p>
+        <div class="recents-events-container">
+          <div>
+            <iframe
+              src="https://youtu.be/o5bfPUMZvws"
+              title="CHOGM Stream"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
+          <div>
+            <iframe
+              src="https://www.youtube.com/watch?v=-Qp0a4jG23A"
+              title="CHOGM Stream"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </div>
```

Comment:
- [ ] Kindly make sure that the youtube iframe wark or you can consider removing them
![Screen Shot 2022-06-25 at 13 22 31](https://user-images.githubusercontent.com/28307960/175771407-413ce843-a2ce-416e-a91c-1d13b46ded67.png)